modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -20,10 +20,12 @@ class DisabilityCompensationPdfMapper # rubocop:disable Metrics/ClassLength
         4 => :convert_date_string_to_format_yyyy
       }.freeze
 
-      def initialize(auto_claim, pdf_data, target_veteran)
+      def initialize(auto_claim, pdf_data, auth_headers, middle_initial)
         @auto_claim = auto_claim
         @pdf_data = pdf_data
-        @target_veteran = target_veteran
+        @auth_headers = auth_headers&.deep_symbolize_keys
+        @middle_initial = middle_initial
+        @veteran_data = JSON.parse(auth_headers['va_eauth_authorization'])&.deep_symbolize_keys
```

Comment:
do we _**need**_ to dive one level deeper? I believe we're only pulling first name, last name and birthdate, which should all exist on the auth headers if I'm understanding correctly. it looks like the method that builds auth_headers has
`va_eauth_firstName`
`va_eauth_lastName`
`va_eauth_birthdate`